OnlineShop.ApiGateway.Ocelot/ocelot.json

```diff
@@ -0,0 +1,193 @@
+{
+  "Routes": [
+    // Auth Microservice
+    {
+      "UpstreamPathTemplate": "/api/auth/{everything}",
+      "UpstreamHttpMethod": [ "Post" ],
+      "DownstreamScheme": "https",
+      "DownstreamHostAndPorts": [
+        {
+          "Host": "Auth_service",
+          "Port": 443
+        }
+      ],
+      "DownstreamPathTemplate": "/api/auth/{everything}",
```

Comment:
why did you define specific routes in other microservices but for auth service you used {everything}?